File: Beecrowd/1041 Coordenadas de um Ponto/main.py
entrada = input().split()
X, Y = entrada

X = float(X)
Y = float(Y)
           
if X == 0 and Y == 0:
    print("Origem")
elif X == 0:
    print("Eixo Y")
elif Y == 0:
    print("Eixo X")
if X < 0  and Y < 0:
    print("Q3")
elif X < 0 and Y > 0:
    print("Q2")
elif Y < 0 and X > 0:
    print("Q4")
elif X > 0 and Y > 0:
    print("Q1")

File: Beecrowd/1041 Coordenadas de um Ponto/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	double X, Y;
	
	cin >> X;
	cin >> Y;
	
	if(X == 0 && Y == 0)
		cout << "Origem" << endl;
	else if(X == 0)
		cout << "Eixo Y" << endl;
	else if(Y == 0)
		cout << "Eixo X" << endl;	
	if(X < 0.0  && Y < 0.0)
		cout << "Q3" << endl;
	else if(X < 0.0 && Y > 0.0)
		cout << "Q2" << endl;
	else if(Y < 0.0 && X > 0.0)
		cout << "Q4" << endl;
	else if(X > 0.0 && Y > 0.0)
		cout << "Q1" << endl;
 
    return 0;
}